src/runtime/ExecutionState.h

```diff
@@ -106,6 +106,11 @@ class ExecutionState : public gc {
         m_inStrictMode = inStrictMode;
     }
 
+    void _setLexicalEnvironment(LexicalEnvironment* lexicalEnvironment)
+    {
+        m_lexicalEnvironment = lexicalEnvironment;
```

Comment:
This is a temproary hack before #300 is landed. Or the relevant part could be moved into this patch if needed.